doc/nrf/libraries/networking/nrf_provisioning.rst

```diff
@@ -0,0 +1,162 @@
+.. _lib_nrf_provisioning:
+
+nRF Device provisioning
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF Device provisioning library enables a device to connect to Nordic Semiconductor's provisioning Service.
```

Comment:
Unsure about the capitalization: Device Provisioning? Neither capitalized?

If we are talking about the cloud-side provisioning service, I believe the name will be nRF Cloud Provisioning Service, part of nRF Cloud Security Services. I can check with PMT.

Reply:
Please check the correct name.

Reply:
Just to check, the service we're talking about here _is_ the Cloud service?

Reply:
Yes, the Cloud service